package/upgrade/upgrade_manifests.sh

```diff
@@ -406,6 +406,70 @@ EOF
   wait_rollout cattle-monitoring-system deployment rancher-monitoring-operator
 }
 
+upgrade_logging_event_audit() {
+  # from v1.0.3, logging, event, audit are enabled by default
+  echo "The current version is $UPGRADE_PREVIOUS_VERSION, will check Logging Event Audit upgrade manifest option"
+
+  if test "$UPGRADE_PREVIOUS_VERSION" = "v1.0.3"; then
+    echo "Logging Event Audit: start to upgrade manifest"
+
+    # prepare a malformed yaml file, make sure it is effectively replaced
+    echo "to-be-replaced-file" > rancher-logging.yaml
+
+    # reuse frame work to generate yaml file
+    upgrade_managed_chart_from_version $UPGRADE_PREVIOUS_VERSION rancher-logging rancher-logging.yaml
+
+    echo "Apply resource file"
+
+    kubectl apply -f ./rancher-logging.yaml
+
+    # wait for managedchart to be applied
+    sleep 50
```

Comment:
The feature looks good, is it possible to use the `wait_managed_chart` call here?